src/TailwindCssPreset.php

```diff
@@ -30,10 +30,10 @@ public static function installAuth()
     protected static function updatePackageArray(array $packages)
     {
         return array_merge([
-            'laravel-mix' => '^4.0.14',
+            'laravel-mix' => '^5.0',
```

Comment:
This should be `^5.0.1` per the latest `package.json` in [laravel/laravel](https://github.com/laravel/laravel/blob/master/package.json)